fontra2fontir/src/source.rs

```diff
@@ -188,6 +191,36 @@ impl Source for FontraIrSource {
     }
 }
 
+/// serde type used to load font-data
+#[derive(Debug, Clone, Deserialize)]
+struct FontData {
+    #[serde(rename = "unitsPerEm")]
+    _units_per_em: u16,
+    #[serde(default)]
+    _axes: Vec<FontraAxis>,
+}
+
+impl FontData {
+    fn from_file(p: &Path) -> Result<Self, Error> {
+        let raw = fs::read_to_string(p).map_err(Error::IoError)?;
+        serde_json::from_str(&raw).map_err(|e| Error::ParseError(p.to_path_buf(), format!("{e}")))
+    }
+}
+
+#[derive(Debug, Clone, Deserialize)]
+struct FontraAxis {
+    _name: String,
```

Comment:
why all the underbars? Is this just to avoid dead code warnings? In general I'd prefer to just mark items as `#[allow(dead_code)]` while things are in progress, but 🤷

Reply:
For dead code. It becomes live in the follow-on so for this PR I'm inclined to leave the _'s